src/services/ombd-service.js

```diff
@@ -0,0 +1,44 @@
+import 'es6-promise';
+import 'isomorphic-fetch';
+
+export default class OmbdService {
+  constructor(key) {
+    this.key = key;
+  }
+
+  async search(title, initPage = 1, numPages = 1) {
+    // prettier-ignore
+    const url = `https://www.omdbapi.com?apikey=${this.key}&s=${title}`
```

Comment:
woof! Missing semicolon
This is because prettier is ignored for this line that it's not added automatically